### DISCUSSION TITLE
mhig_eu_au_k1555

### DISCUSSION THREAD

**Discussion by avo-NL** [Mr-MIBonk/M.I.B._More-Incredible-Bash#331] [General]
hello,

I followed mibhelper tutorials but I can't update the mapping. I passed for the software.

I have version MHIG_EU_AU_K1555 (after the update MHIG_EU_AU_S1545)
activation key 0210001e

And I downloaded version P330_N60S5MIBH3_EU of the 2023 maps.

The MMI tells me that it cannot find any valid data.

Can you help me ?

Thanks in advance

**Comment by aledepaoligampel** [comment]
Hi @avo-NL, can you solved the problem? I have the same issue.
